docs/gettingstarted.rst

```diff
@@ -1,205 +1,65 @@
-to set up a connection to Kubernetes infrastructure, so in order
-to use reana-cluster one needs to have kubectl installed
-and configured properly.
+Are you looking at installing and deploying REANA cluster locally on your laptop?
 
+1. Install Minikube following `minikube installation guide <https://kubernetes.io/docs/getting-started-guides/minikube/#installation>`_.
 
-Test reana-cluster using Minikube
----------------------------------
-
-Easiest way to test reana-cluster without dedicating time to
-setting up a real Kubernetes infrastructure is to use
-`Minikube <https://kubernetes.io/docs/getting-started-guides/minikube/>`_,
-which is a spins up a working Kubernetes deployment in a virtual machine.
-
-Once you have installed kubectl and Minikube, start Minikube by running:
+2. Start Minikube:
 
 .. code-block:: console
 
    $ minikube start --kubernetes-version="v1.6.4"
-   ...
-   minikube: Running
-   cluster: Running
-   kubectl: Correctly Configured: pointing to minikube-vm at 192.168.99.100
 
-.. note::
-   Currently REANA supports only Kubernetes v1.6.4
-
-As seen from the output, Minikube startup routines should
-configure kubectl to interact with the newly created
-virtual machine, but it best to test that kubectl is indeed
-configured properly:
+3. Install REANA-Cluster sources. You probably want to use a virtual environment:
 
 .. code-block:: console
 
-   $ kubectl get all
-   NAME             CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
-   svc/kubernetes   10.0.0.1     <none>        443/TCP   3d
-
-
-Install reana-cluster cli tool
-------------------------------
-
-reana-cluster command line interface tool is not yet released
-in PyPI, so install it from our GitHub repotory:
-
-.. code-block:: console
-
-   $ pip install \
-     -e 'git+https://github.com/reanahub/reana-cluster.git@master#egg=reana-cluster'
-
-
-Initialize a REANA cluster
---------------------------
-
-Main function of reana-cluster command line tool is to
-initialize a working REANA cluster, ready to run workflows
-you submit to it using reana-client.
-
-In order to achieve this reana-cluster needs to know how
-REANA cluster should be set up; e.g. what versions of REANA
-components should be deployed and how the configuration of each
-component should be set up.
-
-reana-client expects to get information via REANA cluster
-specification file.
+   $ mkvirtualenv reana-cluster
+   $ pip install -e 'git+https://github.com/reanahub/reana-cluster.git@master#egg=reana-cluster'
 
-The specifications file is written in YAML syntax, but since
-reana-cluster is still work-in-progress the structure of
-REANA specifications file might change rapidly.
-We therefore suggest that you download our default
-configuration file from the root of our GitHub repository:
+4. Configure REANA cluster. You can download default ``reana-cluster.yaml`` configuration file:
```

Comment:
Won't `reana-cluster.yaml` be present in the package itself?

Reply:
Yes, but now it is not 😄

Reply:
Should we add this comment here as well?
https://github.com/reanahub/reana-cluster/pull/38/files#diff-bb7aeded5e46a9d6e405411c85a2d47dR82

It might clutter the getting started a bit, but it would also make sure that the user understands where to get the latest working `reana-cluster.yaml`. On the other hand `reana-cluster.yaml` coming with the package contains references to images that we know to be deployable with `reana-cluster.yaml`. Of course if we make a mistake in `reana-cluster.yaml` or want to introduce some change that requires editing of `reana-cluster.yaml` in a REANA component (e.g. add an environment variable that affects configuration), we would have to release a new version of `reana-cluster`.

Reply:
I plan to add `reana-cluster*yaml` to the Pyhton package, so we shall be able to simplify getting started.  We could then enrich the manual to say that REANA 3.4.7 may not work with 3.5.0 spec files etc.  The Getting started should stay as simple as possible...